include/seqan3/range/views/minimiser.hpp

```diff
@@ -358,8 +364,11 @@ class minimiser_view<urng_t>::window_iterator
             window_values.push_back(*window_right);
             std::ranges::advance(window_right,  1);
         }
+
         window_values.push_back(*window_right);
-        minimiser_value = *(std::min_element(std::begin(window_values), std::end(window_values)));
+        auto deque_it = std::min_element(std::begin(window_values), std::end(window_values), less_or_equal_compare);
```

Comment:
This was already done in another PR and was present after I rebased now.

> Are some tests failing? (I'd expect it because we change the behaviour)

They shouldn't. I changes the text in the test to reflect the new behaviour. I'll see what happends after the rebase.

Reply:
There are some failures in the `view_minimiser_hash_test `.
Because we now always take the right most minimizer, there is only one minimizer instead of 3.